util/linuxfw/nftables_runner.go

```diff
@@ -953,6 +1102,29 @@ func (n *nftablesRunner) DelSNATRule() error {
 	return nil
 }
 
+func cleanupChain(logf logger.Logf, conn *nftables.Conn, table *nftables.Table, hookChainName, tsChainName string) {
```

Comment:
this function needs a docstring now as what it is using `hookChainName` and `tsChainName` for could do with some explanation

Reply:
Updated